experimental/unitsconversion/src/provider.rs

```diff
@@ -24,13 +24,15 @@ use zerovec::{VarZeroVec, ZeroVec};
 /// </div>
 pub struct Baked;
 
+include!("../../../provider/datagen/tests/data/baked/macros.rs");
```

Comment:
I made this use the stub data until we can move it into experimental. It's singleton anyway.